python/lsst/ts/fbs/utils/auxtel/surveys.py

```diff
@@ -77,17 +103,30 @@ def generate_image_survey(
         additional_notes=[(target.target_name.split("_")[0], 32)],
     )
 
-    sequence = [ObservationArray(n=1) for i in range(len(target.filters))]
+    # Set up a sequence of nexp exposures per filter
+    # sequence_len = target.nexp * len(target.filters)
```

Comment:
Please don't leave commented code behind...

Reply:
I'll take this out .. hopefully we can update the rubin_scheduler at the summit shortly, and then revert to what was commented out.